app/views/repositories/_repository.html.erb

```diff
@@ -0,0 +1,38 @@
+<div id='repository-card' class='p-4 border-solid border-2 rounded-xl mb-3'>
+  <div class='flex justify-between items-start mb-4'>
+    <div class='flex flex-col gap-2'>
+      <%= link_to repository.link, target: "_blank" do %>
+        <h2 class="font-semibold text-lg m-0"><i class="fab fa-github mr-2"></i><%= repository.link %></h2>
+      <% end %>
+      <span class="d-block text-dark"><%= simple_format repository.description %></span>
+    </div>
+    <div class='flex gap-4 items-center'>
+    <% if repository.issues.present? %>
+      <div class='flex gap-1 items-center text-gray-400' title="<%= "#{t("repositories.index.issues")}: #{repository.issues}" %>">
+        <%= inline_svg_tag 'issues.svg', class: 'h-4 w-4' %>
+        <span id='issues-tag' class='font-semibold'><%= repository.issues_formatted %></span>
+      </div>
+    <% end %>
+    <% if repository.stars.present? %>
```

Comment:
Why does this check when a repo will not have stars?

Reply:
I decided to put these conditions for cases when for some reason a repository does not contains issues  count or stars count.

Reply:
Ok, I got the idea, but if we get at this point and the star, issues, etc are nil, we probably have a problem in some other part of the system, this guard might obfuscate the error